installer/cloudflow-environment/values.yaml

```diff
@@ -26,7 +26,7 @@ kafka:
     deletePersistentVolumeClaimsOnDelete: false
     useDedicatedNodes: false
     kafka:
-      version: 2.3.0
+      version: 2.3.1
```

Comment:
Why not use Kafka 2.4.0 instead?

Reply:
Yes we can use it. Strimzi 0.16.x (we are on 0.16.2) dropped support for Kafka 2.2.1 and 2.3.0 and added support for 2.4.0.

Reply:
How about we close this PR? I updated to 2.4.0 in https://github.com/lightbend/cloudflow/pull/132/.

Reply:
Fine with me